src/app/shared/header/header.component.html

```diff
@@ -0,0 +1,17 @@
+<nav class="navbar navbar-expand-lg bg-light">
+  <a class="navbar-brand" routerLink="/home">
+    <img src="/../assets/images/cola-logo.png" alt="logo" width="150px" height="50px">
+
+  </a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup"
+          aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+    <div class="navbar-nav">
+      <a class="nav-item nav-link" routerLinkActive="active" routerLink="home">Home</a>
+      <a class="nav-item nav-link" routerLinkActive="active" routerLink="contact">Contact</a>
```

Comment:
Create an array with current links and render it with *ngFor

Reply:
Done